packages/flutter/test/foundation/memory_allocations_test.dart

```diff
@@ -160,20 +162,97 @@ void main() {
     _checkSdkHandlersNotSet();
     expect(ma.hasListeners, isFalse);
   });
+
+  testWidgets('State dispatch events in debug mode', (WidgetTester tester) async {
```

Comment:
This test probably belongs under `/test/widgets` and not foundation.

Reply:
Same below, the element test also belongs to `test/widgets`. The Layer and RO test belongs to `test/rendering`.

Reply:
done